python/sparknlp/pretrained.py

```diff
@@ -0,0 +1,45 @@
+import sparknlp.internal as _internal
+from sparknlp.base import DocumentAssembler
+from sparknlp.annotator import *
+from pyspark.ml.wrapper import JavaModel
+from pyspark.sql import DataFrame
+from sparknlp.base import LightPipeline
+
+
+class ResourceDownloader(object):
+
+    @staticmethod
+    def downloadModel(reader, name, language, remote_loc=None):
```

Comment:
Can we also add another field called `source` which represents the source of the dataset which we built the model? For instance, we will have multiple Lemma in Italian one from DXC, one from UD, one from the famous GitHub repo for Lemma Dictionaries, etc.
This way, we can have `source` default to one of these models, and the user can choose whether they wan POS Tagger in English based on regular format or the one from the UD treebank.

Reply:
For now we could name it just differently, once we get to a point to multiple sources per public model we can re-consider